proposals/type-aliases.md

```diff
@@ -221,6 +289,7 @@ fun throws(): Empty = ...       // Error: return type Nothing should be specifie
 ```
 
 ### Type aliases as values
```

Comment:
What wording we use in the code is hardly a justification for saying something ambiguous in public docs :) "Value" is something rather unclear in this context, and, unlike the code, "expression" here means only one thing